.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# Code owners file.
+# This file controls who is tagged for review for any given pull request.
+
+# For anything not explicitly taken by someone else:
+*       @g-easy @jmacd @maxgolov @reyang @rnburn @tigrannajaryan
```

Comment:
Optional: would you prefer to manage this via github and set the owners to: @open-telemetry/cpp-approvers @open-telemetry/cpp-maintainers

Reply:
I guess having individuals' name here gives additional benefit (e.g. email filter rules, people can opt-out even if they are approver/maintainer)?
I checked other repo and they seem to use individual names:
* https://github.com/open-telemetry/opentelemetry-js/blob/master/.github/CODEOWNERS
* https://github.com/open-telemetry/opentelemetry-python/blob/master/.github/CODEOWNERS

I don't have strong opinion here, will go ahead and merge first, would be happy to make changes later on.

Reply:
I'm happy with this as-is. Thank you, @reyang.